VideoWeb/VideoWeb/ClientApp/src/app/shared/service-down-banner/service-down-banner.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-!-margin-top-3">
+  <div
+    class="govuk-notification-banner govuk-notification-banner--warning"
+    role="region"
```

Comment:
any chance we could fix the sonar issue?

Reply:
Will do, I went with the exact snippet from GDS